actors/miner/src/policy.rs

```diff
@@ -15,10 +15,25 @@ use fvm_shared::sector::{
     RegisteredPoStProof, RegisteredSealProof, SectorQuality, SectorSize, StoragePower,
 };
 use fvm_shared::version::NetworkVersion;
+use lazy_static::lazy_static;
 
 use super::types::SectorOnChainInfo;
 use super::{PowerPair, BASE_REWARD_FOR_DISPUTED_WINDOW_POST};
 
+/// Precision used for making QA power calculations
```

Comment:
shouldnt these be in the policy object?

Reply:
Interesting question.  These fall into the category of "parameters" that are effectively constants in that that don't make a lot of sense to change which I believe @Stebalien was identifying in the review of the policy PR.  For example there's no good reason I can think of to run a devnet with this value changed.  So I think the answer is no.